src/services/objection.session.service.ts

```diff
@@ -4,9 +4,10 @@ import { SessionKey } from "ch-node-session-handler/lib/session/keys/SessionKey"
 import { SignInInfoKeys } from "ch-node-session-handler/lib/session/keys/SignInInfoKeys";
 import { UserProfileKeys } from "ch-node-session-handler/lib/session/keys/UserProfileKeys";
 import { IAccessToken, ISignInInfo, IUserProfile } from "ch-node-session-handler/lib/session/model/SessionInterfaces";
-import { OBJECTIONS_SESSION_NAME, SESSION_COMPANY_PROFILE } from "../constants";
+import {OBJECTIONS_SESSION_NAME, SESSION_COMPANY_PROFILE, SESSION_OBJECTION_CREATE} from "../constants";
 import ObjectionCompanyProfile from "../model/objection.company.profile";
 import ObjectionSessionExtraData from "../model/objection.session.extra.data";
+import {ObjectionCreate} from "../modules/sdk/objections";
```

Comment:
Space before and after {} 
Running eslint with the --fix argument should correct all of these in one go for you

Reply:
Spaces added.